### PR TITLE
Upgrade client for support API v2

### PR DESCRIPTION
**Pull request by pleum**
Upgrade client for support API v2

Fixes: https://github.com/jrwren/domasimu/issues/2

### CODE REVIEW

File: cmd/domasimu/main.go [hunk 1]
```diff
@@ -6,14 +6,17 @@
...
 	"flag"
 	"fmt"
 	"os"
 	"path/filepath"
+	"strconv"
 	"strings"
 
 	"github.com/BurntSushi/toml"
-	"github.com/pearkes/dnsimple"
+	"github.com/dnsimple/dnsimple-go/dnsimple"
```
Comment (jrwren) [inline cmd/domasimu/main.go:18]: Will you please update the .travis.yml to reflect this package change so that tests will fetch the right package?

**Review by jrwren — CHANGES_REQUESTED** [review]
This looks great! Huge Thanks To You.

**Review by jrwren — APPROVED** [review]

File: cmd/domasimu/main.go (hunk 1 above)
Comment (pleum) [reply cmd/domasimu/main.go:18]: Sure! 😁

**Comment by jrwren** [comment]
HUGE thanks! This is great!

**Pull request merged**